tasks.py

```diff
@@ -707,8 +707,8 @@ def check_schema(context, api_version=None):
 )
 def unittest(
     context,
-    cache_test_fixtures=False,
-    keepdb=False,
+    cache_test_fixtures=True,
+    keepdb=True,
```

Comment:
Should CI now do `--no-cache-test-fixtures` and/or `--no-keepdb`? Does it have any measurable performance impact either way?

Reply:
I am a little confused. CI was running with `--cache-test-fixtures` and `--keepdb` in the first place. Why would we want to opt out?

Reply:
Huh, that seems counterintuitive to me. What's the point of caching the test fixtures and keeping the DB when each CI test execution runs in isolation?

Reply:
I can explicitly add `--no-cache-test-fixtures` and `--no-keepdb` to the CI